mne_bids/tests/test_io.py

```diff
@@ -31,6 +32,23 @@ def test_read_raw():
     with pytest.raises(ValueError, match='file name extension must be one of'):
         _read_raw(f)
 
+    # Check that KIT and BTI data is automatically read
+    # KIT
+    base_path = op.join(op.dirname(mne.__file__), 'io')
+    data_path = op.join(base_path, 'kit', 'tests', 'data')
+    raw_fname = op.join(data_path, 'test.sqd')
+    hpi_fname = op.join(data_path, 'test_mrk.sqd')
+    electrode_fname = op.join(data_path, 'test_elp.txt')
+    headshape_fname = op.join(data_path, 'test_hsp.txt')
+    _read_raw(raw_fname, electrode=electrode_fname, hsp=headshape_fname,
+              hpi=hpi_fname)
```

Comment:
why are these not covered by `write_raw_bids`?

Reply:
the tests only covered this function for Elekta data. The other `if..else` statements for EDF/BDF, BTi and KIT data weren't covered.
`write_raw_bids` doesn't cover this function because it takes a `raw` object, which has to have already been read. The only reason we had any real coverage for this function is because of `mne_bids_raw_to_bids.py`

Reply:
I would then modify `test_commands.py`. Although I think it's probably worth waiting on this a bit because in #135 (ps: you are welcome to review it :p) this function is being used. So coverage will automatically increase once that's merged.